package.json

```diff
@@ -68,6 +69,7 @@
     "babel-eslint": "^10.1.0",
     "babel-loader": "^8.1.0",
     "chalk": "^4.1.1",
+    "concurrently": "^6.0.2",
```

Comment:
We have `npm-run-all` in the deps already - this can run tasks in parallel (see the `npm build` task).

Reply:
Good catch!